src/main/java/se/sundsvall/supportmanagement/service/mapper/NotificationMapper.java

```diff
@@ -81,13 +81,13 @@ public static Notification toNotification(final Event event, final ErrandEntity
 			.withType(event.getType().getValue())
 			.withExpires(event.getExpires())
 			.withModified(event.getCreated())
-			.withCreated(event.getCreated());
+			.withCreated(event.getCreated())
+			.withCreatedBy(executingUser)
+			.withOwnerId(errandEntity.getAssignedUserId());
 		Optional.ofNullable(owner)
-			.ifPresent(o -> notification.withOwnerId(o.getLoginName())
-				.withOwnerFullName(o.getFullname()));
+			.ifPresent(o -> notification.withOwnerFullName(o.getFullname()));
```

Comment:
Might want to use the normal setter if no chaining calls? This might be a little confusing otherwise, with no context, this might be a record with lomboks @With annotation.

same at row 90